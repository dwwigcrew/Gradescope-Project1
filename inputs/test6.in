6
3
237456
10
555.55