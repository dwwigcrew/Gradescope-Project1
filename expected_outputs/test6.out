
Welcome to the ENEE140 pseudo-random number generator!
1: Print RND_MAX
2: Generate uniformly-distributed positive integers
3: Generate uniformly-distributed positive integers, up to a given limit
4: Generate uniformly-distributed integers, from a given range
5: Generate uniformly-distributed floats, from a given range
6: Generate exponentially-distributed floats

Enter your choice (1-6):
Select the algorithm to use.

Enter your choice (1-3):
Select the seed for the random number generator:
How many numbers should I generate:
Enter the mean of the distribution: 990.18 835.33 420.29 383.62 825.43 256.81 700.21 691.16 639.11 188.73
